packages/cli/src/oclif/ZapierBaseCommand.js

```diff
@@ -143,11 +143,15 @@ class ZapierBaseCommand extends Command {
   }
 
   startSpinner(message) {
-    startSpinner(message);
+    if (this._shouldPrintData()) {
```

Comment:
were you seeing issues before adding this? my understanding is that `ora` uses `stderr`, so it doesn't actually interfere with formatting. 

For instance: 

![](https://zappy.zapier.com/E3B141E2-D2E2-4311-AF6C-50AB4AAACB8C.png)

Not a hard blocker, but the spinners add a much better UX (rather than the command seeming to hang), so I want to make sure they're shown as much as possible.

Reply:
Good catch! I didn't know spinners go to stderr. Undid it in a4ca9e7.